lib/tmuxinator/cli.rb

```diff
@@ -344,5 +355,25 @@ def doctor
       say "Checking if $SHELL is set ==> "
       yes_no Tmuxinator::Doctor.shell?
     end
+
+    # This method was defined as something of a workaround...  Previously
+    # the conditional contained within was in the executable (i.e.
+    # bin/tmuxinator).  It has been moved here so as to be testable. A couple
+    # of notes:
+    # - ::start (defined in Thor::Base) expects the first argument to be an
+    # array or ARGV, not a varargs.  Perhaps ::bootstrap should as well?
```

Comment:
Using a regular parameter does seem cleaner and, unless I'm missing something, I can't think of a scenario where the splat would be useful. Seems like after removing the two `*`s, everything should continue to work? If not, now it'll be easy enough to test. ;)

Reply:
Done